tests/functional/Chrome/ChromeDriverTest.php

```diff
@@ -33,18 +33,36 @@ protected function tearDown(): void
         }
     }
 
-    public function testShouldStartChromeDriver()
+    /**
+     * @dataProvider provideDialect
+     * @param bool $isW3cDialect
+     */
+    public function testShouldStartChromeDriver($isW3cDialect)
     {
-        $this->startChromeDriver();
-
+        $this->startChromeDriver($isW3cDialect);
         $this->assertInstanceOf(ChromeDriver::class, $this->driver);
         $this->assertInstanceOf(DriverCommandExecutor::class, $this->driver->getCommandExecutor());
 
+        // Make sure actual browser capabilities were set
+        $this->assertNotEmpty($this->driver->getCapabilities()->getVersion());
+        $this->assertNotEmpty($this->driver->getCapabilities()->getCapability('goog:chromeOptions'));
```

Comment:
This shows the fix - this would be failing before. `$this->driver->getCapabilities()` was containing only the capabilities used for creation, not the actual ones from the browser.